assets/js/screens/Collection/Collection.jsx

```diff
@@ -84,15 +102,35 @@ const ScreensCollection = () => {
                 <div className="columns">
                   <div className="column is-two-thirds">
                     <h1 className="title">{data.collection.name || ""}</h1>
+                    <span
```

Comment:
Here can we can add a little bit of horizontal spacing between the tags?  5px maybe?